tests/course_mock_data

```diff
@@ -16,34 +21,107 @@ function start() {
 
 // Increase Limits so couchapp push works correctly
 function increaseLimits() {
```

Comment:
either there should be code to increase limit or this function should not be there.

Reply:
increaseLimits function had been removed